### PR TITLE
[stable10] Make the changes from doc fix persistant 

### PR DESCRIPTION
**Pull request by nickvergessen**
Backport of #496

### CODE REVIEW

File: config/config.sample.php [hunk 1]
```diff
@@ -435,7 +435,7 @@
  * Both minimum and maximum times can be set together to explicitly define
  * file and folder deletion. For migration purposes, this setting is installed
  * initially set to "auto", which is equivalent to the default setting in
- * ownCloud 8.1 and before.
+ * Nextloud.
```
Comment (LukasReschke) [inline config/config.sample.php]: Typo

Comment (nickvergessen) [inline config/config.sample.php]: fixed

Comment (nickvergessen) [inline config/config.sample.php]: Fixed on master as well https://github.com/nextcloud/server/pull/517

**Comment by MorrisJobke** [comment]
👍

**Comment by rullzer** [comment]
:+1:

**Pull request merged**